app/presenters/api/v2/commodities/commodity_presenter.rb

```diff
@@ -48,8 +48,12 @@ def chapter_id
           commodity.chapter.goods_nomenclature_sid
         end
 
+        def ancestors
+          @ancestors ||= ns_ancestors.select { |ancestor| ancestor.is_a?(Commodity) }
```

Comment:
This is a little confusing. So these are basically all subheadings?

Reply:
Yeh, agreed - though at present STI will return them as Commodities 

I was planning to hoist this up to goods nomenclature in the next week or so and I'll make it check for either at that point (more in the scope of the STI ticket though?)